models/ShoppingCart.js

```diff
@@ -0,0 +1,34 @@
+const { Model, DataTypes } = require("sequelize");
+const sequelize = require("../config/connection");
+
+class shopping_cart extends Model {}
+
+ShoppingCart.init({
```

Comment:
You want the class name to match the name you use for init, so in this case: `class ShoppingCart extends Model{}`